tests/test_root_versioning.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  test_root_versioning.py
```

Comment:
Happy to move this around, however there are other files like `test_key_revocation` for example.